components/AppHeader/UserMenu.tsx

```diff
@@ -15,13 +16,11 @@ const UserMenu: React.FC = () => {
 
   const [state, setState] = React.useState<{
```

Comment:
Since the state is no longer an object, the name of this should have an updated name.